packages/chaire-lib-backend/src/services/auth/userEmailNotifications.ts

```diff
@@ -154,7 +154,7 @@ export const sendConfirmationEmail = async (
         emails.forEach(async (email) => {
             await sendEmail(email, { userConfirmationUrl: { url: options.confirmUrl }, usermail: user.email });
         });
-        console.log('Email sent for account confirmation');
+        console.log('Emails (admins and user) sent for account confirmation');
```

Comment:
En fait, on ne sait pas si c'est admin _and_ user, ça dépend de la config, ça peut être juste le user si on a confirmByUser. Donc `admin and/or user` serait plus juste.

Reply:
Bon, encore mieux, je devrais checker le length de emails, et dire XX Emails (admins and/or user) sent for account confirmation